schedule.json

```diff
@@ -52,14 +52,24 @@
   },
   "v11": {
     "start": "2018-10-23",
-    "maintenance": "2019-04-01",
-    "end": "2019-06-30"
+    "maintenance": "2019-04-16",
```

Comment:
I set 2019-04-16 as the date for maintenance as that is the date of the last planned v11.x release